content/download/package-manager.fr.md

```diff
@@ -0,0 +1,356 @@
+---
+title: Installation de Node.js via le gestionnaire de paquets
+description: Installing Node.js via package manager
+authors: fhemberger, XhmikosR, shadowspawn, vsemozhetbyt, nschonni, wildcard, MrJithil, kasicka, cassidyjames, Trott, richardlau, Qantas94Heavy, pierreneter, 0mp, ThePrez, PoojaDurgad, MaledongGit, Megajin, marc-maurer, yodeyer, geek, sudowork, strawbrary, ryanmurakami, rbnswartz, arkwright, oliversalzburg, mweagle, Mohamed3on, Ginden, kapouer, jperkin, jericopulvera, jedsmith, jasonkarns, sonicdoe, mcollina, fornwall, danbev, naskapal, awochna, AdamMajer, ahmetanilgur, bnb, qbit, AugustinMauroy
+category: télécharger
```

Comment:
The `category` field must remain `download`. That's how we identify categories 🙇

Reply:
Oops same error x'D